### DISCUSSION TITLE
ValueError:  [!] Language chinese-mandarin is not supported for phonemization.

### DISCUSSION THREAD

**Discussion by LinJinghuidev** [coqui-ai/TTS#842] [General Q&A]
tts --text "这是一段端到端语音合成结果" --model_name "tts_models/zh-CN/baker/tacotron2-DDC-GST" --vocoder_name "vocoder_models/universal/libri-tts/fullband-melgan" --out_path /home/tk_nd/project/FastSpeech2-master/output/result/Multy/test.wav

**Comment by erogol** [comment]
I just tried on my local with v0.3.1 and it worked. I don't see any issue.

**Reply by LinJinghuidev** [reply]
I ran out and felt that the sound quality was not delicate enough

**Reply by Lizh-TW** [reply]
@LinJinghuidev 加句號"。"，不然會不知道啥時停止就會導致tts你的文字之後跑出無意義的聲音。
Add a period "。", Otherwise, you will cause tts to make a meaningless sound after your output wav.

**Reply by quzhaojing** [reply]
@Lizh-TW It's very helpful for me。 And Where can i find better trained chinese models for coqui-ai tts?

**Reply by Lizh-TW** [reply]
@CharellKing --model_name "tts_models/zh-CN/baker/tacotron2-DDC-GST is good. Otherwise, you need to train a new one .
